.github/workflows/clippy.yml

```diff
@@ -47,25 +47,25 @@ jobs:
 
     # Run
     - name: Build
-      run: cargo build --tests --features deny-warnings,internal
+      run: cargo build --tests --features internal -D warnings
```

Comment:
Clippy is the only `cargo` command that takes `-D` flags. We have documentation in our book how to do this in GHA: https://doc.rust-lang.org/clippy/continuous_integration/github_actions.html

TL;DR: Add this near the top of the workflow file

```yaml
env:
  RUSTFLAGS: "-Dwarnings"
```

Reply:
That would be better even if the other way did work.